src/Promise.js

```diff
@@ -65,7 +65,7 @@ export class Future extends Core {
 	}
 
 	// catch :: Promise e a -> (e -> b) -> Promise e b
-	catch (r) {
+	['catch'] (r) {
```

Comment:
yup buble does this
```js
Future.prototype.catch = function catch(r) {
    var n = this.near();
    return n === this ? (0, _then3.default)(void 0, r, n, new Future()) : n.catch(r);
};
```

and `function catch(){}` is an error but with the bracket notation you get

```js
Future.prototype['catch'] = function (r) {
    var n = this.near();
    return n === this ? (0, _then3.default)(void 0, r, n, new Future()) : n.catch(r);
};
```
so theres no function name